resources/rest-service/cloudify/migrations/versions/1fbd6bf39e84_4_5_to_4_5_5.py

```diff
@@ -18,6 +37,14 @@
 branch_labels = None
 depends_on = None
 
+resource_tables = [Blueprint.__tablename__, Snapshot.__tablename__,
```

Comment:
this had hardcoded tablenames before, no?
hardcoded is better here (IMO), because what if we actually do change a table name, eg in 4.6? this migration will then stop working, because it will then use the new name which exists in 4.6 but not in this 4.5 migration

Reply:
I agree, I changed it because Pavel mention it in the PR, but I'll change it back

Reply:
Hmm, fair point.